### PR TITLE
API Documentation for Migration UX enhancement feature

### PR DESCRIPTION
**Pull request by cinlloc**

### CODE REVIEW

File: apis/en/source/pages/dto-objects/GenerationStage.rst [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Attributes
+~~~~~~~~~~
+
+The list of attributes for ``generationStage`` are:
+
+	* ``image`` (:ref:`image-object`): the image generated by the generation stage
+	* ``targetFormat`` (:ref:`targetFormat-object`): the target format to generate the image
+	* ``targetFormatUri`` (anyURI): the uri resource of the target format
+	* ``pipelineUri`` (anyURI): the uri of the parent pipeline
+	* ``created`` (dateTime): the date the appliance template is created
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/GenerationStage.rst]: should this be date the image is generated??

File: apis/en/source/pages/dto-objects/GenerationStage.rst [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+	* ``dbId`` (long): the database id of the object
+	* ``digest`` (string): the digest value (used for etag)
+	* ``lastModified`` (dateTime): the last modified date of this object
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/GenerationStage.rst]: the date this object was last modified

File: apis/en/source/pages/dto-objects/Migration.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+~~~~~~~~~~
+
+The list of attributes for ``migration`` are:
+
+	* ``stages``: the list of stages to perform the pipeline (see :ref:`stage-object`)
+	* ``tag`` (string): the tag information
+	* ``name`` (string): the pipeline name
+	* ``status`` (:ref:`status-object`): the status of the pipeline
+	* ``uri`` (anyURI): the uri of the pipeline
+	* ``created`` (dateTime): the date the appliance template is created
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/Migration.rst]: appliance template?

File: apis/en/source/pages/dto-objects/Migration.rst [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+	* ``dbId`` (long): the database id of the object
+	* ``digest`` (string): the digest value (used for etag)
+	* ``lastModified`` (dateTime): the last modified date of this object
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/Migration.rst]: the date this object was last modified

File: apis/en/source/pages/dto-objects/MigrationStage.rst [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+Holds all the meta-data of a stage to perform a migration.
+
+Attributes
+~~~~~~~~~~
+
+The list of attributes for ``migrationStage`` are:
+
+	* ``pipelineUri`` (anyURI): the uri of the parent pipeline
+	* ``created`` (dateTime): the date the appliance template is created
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/MigrationStage.rst]: appliance template? should this be migration object?

File: apis/en/source/pages/dto-objects/MigrationStage.rst [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+	* ``dbId`` (long): the database id of the object
+	* ``digest`` (string): the digest value (used for etag)
+	* ``lastModified`` (dateTime): the last modified date of this object
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/MigrationStage.rst]: see above

File: apis/en/source/pages/dto-objects/Pipeline.rst [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+Attributes
+~~~~~~~~~~
+
+The list of attributes for ``pipeline`` are:
+
+	* ``tag`` (string): the tag information
+	* ``name`` (string): the pipeline name
+	* ``status`` (:ref:`status-object`): the status of the pipeline
+	* ``uri`` (anyURI): the uri of the pipeline
+	* ``created`` (dateTime): the date the appliance template is created
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/Pipeline.rst]: ? same query as above

File: apis/en/source/pages/dto-objects/Pipeline.rst [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+	* ``dbId`` (long): the database id of the object
+	* ``digest`` (string): the digest value (used for etag)
+	* ``lastModified`` (dateTime): the last modified date of this object
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/Pipeline.rst]: see above

File: apis/en/source/pages/dto-objects/PublicationStage.rst [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+Attributes
+~~~~~~~~~~
+
+The list of attributes for ``publicationStage`` are:
+
+	* ``publishImage`` (:ref:`publishImage-object`): the image published by the publication stage
+	* ``publishImageUri`` (anyURI): the uri resource of the published image
+	* ``pipelineUri`` (anyURI): the uri of the parent pipeline
+	* ``created`` (dateTime): the date the appliance template is created
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/PublicationStage.rst]: see above

File: apis/en/source/pages/dto-objects/PublicationStage.rst [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+	* ``dbId`` (long): the database id of the object
+	* ``digest`` (string): the digest value (used for etag)
+	* ``lastModified`` (dateTime): the last modified date of this object
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/PublicationStage.rst]: see above

File: apis/en/source/pages/dto-objects/ScanStage.rst [hunk 11]
```diff
@@ -0,0 +1,22 @@
...
+Holds all the meta-data of the scan stage of a migration.
+
+Attributes
+~~~~~~~~~~
+
+The list of attributes for ``scanStage`` are:
+
+	* ``scannedInstanceUri`` (anyURI): the uri resource of the scannedInstance
+	* ``pipelineUri`` (anyURI): the uri of the parent pipeline
+	* ``created`` (dateTime): the date the appliance template is created
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/ScanStage.rst]: the scan is created?

File: apis/en/source/pages/dto-objects/ScanStage.rst [hunk 12]
```diff
@@ -0,0 +1,22 @@
...
+	* ``dbId`` (long): the database id of the object
+	* ``digest`` (string): the digest value (used for etag)
+	* ``lastModified`` (dateTime): the last modified date of this object
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/ScanStage.rst]: see above

File: apis/en/source/pages/dto-objects/Stage.rst [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+
+Holds all the meta-data of a stage to perform a pipeline.
+
+Attributes
+~~~~~~~~~~
+
+The list of attributes for ``stage`` are:
+
+	* ``pipelineUri`` (anyURI): the uri of the parent pipeline
+	* ``created`` (dateTime): the date the appliance template is created
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/Stage.rst]: see above

File: apis/en/source/pages/dto-objects/Stage.rst [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+	* ``dbId`` (long): the database id of the object
+	* ``digest`` (string): the digest value (used for etag)
+	* ``lastModified`` (dateTime): the last modified date of this object
```
Comment (emuus) [inline apis/en/source/pages/dto-objects/Stage.rst]: see above

**Comment by cinlloc** [comment]
Concerning your remarks, these are due to a bug in our tool generation doc. In fact, all the API doc have the same problem. E.g.: http://docs.usharesoft.com/projects/appcenter-api-guide/en/latest/pages/dto-objects/Nic.html
I suggest to let it go like this and open a new global issue about this problem.
@jfknoepfli, @emuus What do you think?

**Comment by emuus** [comment]
Ok for me

**Pull request merged**